lib/gears/color.lua

```diff
@@ -29,6 +29,9 @@ local pattern_cache
 -- range) or nil if input is incorrect.
 -- @usage -- This will return 0, 1, 0, 1
 -- gears.color.parse_color("#00ff00ff")
+-- @function gears.color
+
+--- Parse a HTML-color, same as gears.color
 function color.parse_color(col)
```

Comment:
Uhm, calling `gears.color("foo")` goes to `gears.color.create_pattern`, not `gears.color.parse_color`. Or do I mis-understand this?